doc/doc-site/docs/off-chain/dss/ci-cd.md

```diff
@@ -142,3 +142,19 @@ DSS_SOURCE_REV=dss-v1.5.0
 TERRAFORM_PROJECT_REV=dss-v1.5.0
 source scripts/hot-upgrade.sh $DSS_SOURCE_REV $TERRAFORM_PROJECT_REV
 ```
+
+## Suspend/resume runner
+
+In order to avoid getting charged for a mostly unused service, you can suspend the runner when you're not using it, and resume it before hot upgrading for example.
+
+To do so, you can use `scripts/suspend-runner.sh` like so:
+
+```bash
+source scripts/suspend-runner.sh
+```
+
+and `scripts/resume-runner.sh` like so:
+
+```bash
+source scripts/resume-runner.sh
```

Comment:
Now that I'm reading this flow, I'm realizing that a user will get blocked in resuming the runner when the want to re-engage with a project:

Because the `engage-project` script sets the config, which chooses which runner to use, but it also calls the runner and so it'll error out if the runner is paused. I think the `resume-runner` script needs to accept an argument for the project